### ISSUE TITLE
Manager - Content Media - Unable to save Meta data (alt text for example)

### ISSUE THREAD

**Issue by hedronn** [PiranhaCMS/piranha.core#1764]
Just checking if anyone else is experiencing this behavior on the latest release?

Attempting to update Media Meta (Alt text) fails on Save - Status 400 | piranha.min.js - POST /manager/api/media/meta/save 400 anyone else seeing this error in the Manager - Content Media - Meta Tab when attempting to Save?

Piranha.AspNetCore.Identity	10.0.0	
Piranha.Azure.BlobStorage	10.0.0	
Piranha.Data.EF			10.0.0	
Piranha.Manager			10.0.0	
Piranha.Manager.TinyMCE	10.0.0	
Piranha.WebApi			10.0.0

**Comment by tidyui** [comment]
Bug confirmed. When we updated the security of the manager it seems this POST operation was missed which means it isn’t supplying the anti forgery header to the server.

**Comment by hedronn** [comment]
Thanks for the bug confirmation Håkan. Nice work on getting version 10 out for .NET 6 so fast, my project upgrade went smoothly even with my custom build of Piranha CMS.
